ios/CapacitorCordova/CapacitorCordova/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--
+   PrivacyInfo.xcprivacy
+   CapacitorCordova
+
+   Created by Steven Sherry on 3/6/24.
+   Copyright (c) 2024 jcesarmobile. All rights reserved.
+-->
```

Comment:
I don't want copyright of this 😓 

This is the actual cordova file, if that helps, instead of being empty it includes some values set to false and some empty arrays

Reply:
Ah the copyright must the default for the project, I'm guessing you were the one who initially created it. I'll add the empty stuff like you suggested for both Capacitor and Cordova across both this pr and the 5.x pr.

Reply:
looks like I missed the link
https://github.com/apache/cordova-ios/blob/master/CordovaLib/PrivacyInfo.xcprivacy

Reply:
No worries I found it pretty easy